MetabolicAtlas/MetabolicAtlas
# fix: update footer

**Related issue(s) and PR(s)**  
This PR closes #1335 .



**Type of change**  

 
**List of changes made**  

- Update name of department to "Department of Life Sciences"
- Add link to Contact us section in About page

**Screenshot of the fix**  

![image](https://user-images.githubusercontent.com/123460937/231248463-c4884b75-0b94-4008-9a15-4f5c42325bbf.png)


**Testing**  

- Check that the footer looks okay in multiple browsers and screen sizes
- Check that the link directs the user to `http://localhost/about/platform#contact-us`

**Further comments**  

"My code meets the Acceptance Criteria" has not been checked because I haven't been able to check that the changes don't affect the deployment and don't reduce the performance. But probably they shouldn't be affected by this minor change.

**Definition of Done checklist**
